Source/projections/MissingProjectionForType.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { Exception } from '@dolittle/rudiments';
+import { EventType } from '@dolittle/sdk.artifacts';
+
+/**
+ * Exception that is thrown when there is no projection for a specific event type.
+ */
+export class MissingProjectionForType extends Exception {
+    constructor(eventType: EventType) {
+        super(`Missing projection for '${eventType}'`);
+    }
+}
```

Comment:
This feels a bit of. Is it related to a missing callback or on-method ?

Reply:
Renamed, and added projectionId to the mix